src/fuzzy-picker.js

```diff
@@ -130,7 +138,7 @@ export default class FuzzyPicker extends React.Component {
               {this.props.label}
             </span>
             <span className="fuzzy-picker-instructions">
-              <span><strong>tab</strong> or <strong>↑↓</strong> to navigate</span>
+              <span><strong>tab</strong> or <strong>??</strong> to navigate</span>
```

Comment:
I think it'd be desirable to keep the up and down arrows for navigation? I might be misreading the diff, let me know if I'm off base.

Reply:
Oh sorry. Not sure what happened there! My text editor must have it's own ideas on that!